design-book/src/writing-style.md

```diff
@@ -0,0 +1,24 @@
+# Writing Style
+
+## Common Rules
+
+- Bevy uses American English.
+- Text should be clear and to the point.
+- Prefer plain and simple language and avoid contractions and abbreviations.
+- Language should be targeted at the lowest level of technical expertise needed to clearly understand an explanation.
+- Entity, component, resource, system, event and asset are the critical technical vocabulary for all Bevy users to learn, no matter what their role is.
+- Do not use English contractions like aren’t, can’t, etc.
+- Single character identifiers, like X, Y, Z, R, G, B, etc. are always capitalized.
+- Do not use implementation specific language.
+  - Don't use "int" or "struct"
+  - Instead use "Number" or "Data"
+- Don't use direct language, such as you. This sounds like an accusation towards the user.
```

Comment:
Can be cut; I don't necessarily agree.

Reply:
"Don't use direct language, such as you. This sounds like an accusation towards the user."
I really do think we should keep this one, is there any reason not to?

Reply:
Just a matter of style. I like the tone of Rust's error messages, and they don't follow this.

Reply:
Thats fair